internal/ethapi/api.go

```diff
@@ -2063,28 +2063,33 @@ type CallBundleArgs struct {
 	BaseFee    *big.Int          `json:"baseFee"`
 }
 
-//
-// CallBundle will simulate a bundle of transactions on top of
-// the most recent block. Partially follows flashbots spec v0.5.
+// TODO: this is a rewrite of the below CallBundle API, I am attempting to recreate the logic but skip on all
```

Comment:
Great comment adds in this function! Could we revert this top level message back to what it was before to give flashbots attribution + references the spec version